### DISCUSSION TITLE
How to find out which materials are available?

### DISCUSSION THREAD

**Discussion by wiesehahn** [tylermorganwall/rayshader#262] [Q&A] [answered]
Lets say I want to render points, there is the option to define material and related arguments. Is there a list of aviailable materials somewhere (maybe in rayrender documentation) ?

```
render_highquality(point_radius = 6, clamp_value=10, min_variance = 0,
                  sample_method = "sobol_blue", samples = 128,
                  point_material = rayrender::glossy, 
                  point_material_args = list(gloss = 0.5, reflectance = 0.2))
```

**Comment by tylermorganwall** [comment, marked as answer]
See the materials section here for the functions:

http://www.rayrender.net/reference/index.html

Also, if you misspell any arguments in `point_material_args` (or just include a nonsense argument), the function will print out all the available arguments that you can use for that material.

**Reply by wiesehahn** [reply]
Thanks, nice documentation, somehow didn't come across this before.
